components/__design__/logos.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import Image from 'next/image';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const JSLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+       <div>
```

Comment:
I feel that the indentation is wrong here, I assume you edited this file directly on GitHub UI?

Could you pull your latest changes and run `npm run format` to ensure that the file is properly formatted?

Reply:
Will do, and yes, I did edit it directly on Github. Sorry about making this so complicated.

Reply:
No worries! Completely understandable 😄